esp/applications/sql2ecl/esp.yaml

```diff
@@ -4,6 +4,16 @@ esp:
    daliServers: dali
    loadDaliBindings: false
    auth: ldap
+   authDomains:
+     authDomain:
```

Comment:
@afishbeck the code below is treating authDomain as a list, but the yaml are defining it as an object.  Is it purely a backward compatibility feature, or should the yaml reflect that it is a list?

Reply:
Good catch.  @wangkx can correct me if I'm wrong, but believe having multiple authDomains in baremetal came down to the fact that we ran multiple services on multiple ports in one process.  Each could have a different authDomain configuration.

In containerized mode we support only one service per esp instance and I think we can simplify this by getting rid of the outer tag "authDomains" and keeping "authDomain" as a single object.

We can then change application_config.cpp to treat it as a single object rather than multiple.